app/controllers/GreeterServiceClientInjector.scala

```diff
@@ -0,0 +1,20 @@
+package controllers
+
+import akka.actor.ActorSystem
+import akka.grpc.GrpcClientSettings
+import akka.stream.Materializer
+import io.grpc.examples.helloworld.GreeterServiceClient
+import javax.inject.{Inject, Provider}
+
+import scala.concurrent.ExecutionContext
+
+class GreeterServiceClientInjector @Inject()(implicit sys: ActorSystem, mat: Materializer) extends Provider[GreeterServiceClient] {
+
+  override def get(): GreeterServiceClient = {
+    implicit val ec: ExecutionContext = sys.dispatcher
+    // TODO get settings from the actorsystem configuration
+    // https://github.com/akka/akka-grpc/pull/254
+    val settings: GrpcClientSettings = GrpcClientSettings("localhost", 8080)
```

Comment:
How about the service discovery? I guess we will inject `SimpleServiceDiscovery` on the provider, but then the next question is, do we want it to depend on akka-management?

And where do we want to have those generators? Do we want to have it as side projects of akka-grpc? Or should it be Play/Lagom plugins?

Reply:
I think akka-grpc will depend on akka-management (@cbatey)

Reply:
Since right now akka-grpc is the component that is moving most quickly I think it makes sense to have the generator in akka-grpc. Eventually, as the generation API stabalizes, we will probably move them to separate Play/Lagom plugins. Adding them via the `akkaGrpcExtraGenerators` key (for sbt, and equivalents for gradle/maven) should make that relatively smooth.

Reply:
Indeed, no way around it. Which is actually good and for general usage, the service discovery can lookup the host and port from a config file as well.

Would you make the code generator as part of akka-grpc or a Lagom/Play thing?